cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -386,6 +388,25 @@ def _mapped_single_loop(self, repetition_id: Optional[str] = None) -> 'cirq.Circ
             circuit, self.parent_path, bindable_keys=self._extern_keys
         )
 
+    @cached_property
+    def _mapped_repeat_until(self) -> Optional['cirq.Condition']:
+        repeat_until = self.repeat_until
+        if not repeat_until:
+            return repeat_until
+        if self.measurement_key_map:
+            repeat_until = protocols.with_measurement_key_mapping(
+                repeat_until, self.measurement_key_map
+            )
+        if self.param_resolver:
+            repeat_until = protocols.resolve_parameters(
```

Comment:
@daxfohl You probably saw this in the CI checks, but just in case: the [code coverage check is complaining](https://github.com/quantumlib/Cirq/actions/runs/12572595382/job/35051696880#step:7:1657) about the 3 lines in this clause not being covered in tests.